### PR TITLE
test(toast): integrated and added test cases for component

### PR DESCRIPTION
**Pull request by mJarsater**

### CODE REVIEW

File: src/components/Toast.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import { useState } from "react";
```
Comment (timrombergjakobsson) [inline src/components/Toast.tsx:1]: Did we mention this on checkin, is this the way we want to integrate and test our components?

Comment (mJarsater) [reply src/components/Toast.tsx:1]: I think so, we said that we would create separate files for each component and then import them in the main one.

Comment (timrombergjakobsson) [reply src/components/Toast.tsx:1]: Ok, so for every Jira ticket: [Component]: manual testing. We do this?

Comment (mJarsater) [reply src/components/Toast.tsx:1]: Yes, and test the events and the stuff that are mentioned in the ticket.

Comment (timrombergjakobsson) [reply src/components/Toast.tsx:1]: Ok gotcha!

**Pull request merged**